libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/model/OfflineRouteError.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.navigation.route.onboard.model
+
+import com.google.gson.annotations.SerializedName
+
+internal data class OfflineRouteError(
+    val status: String,
+    @SerializedName("status_code") val statusCode: Int,
```

Comment:
let's move `val statusCode: Int` to the next line. 
in `BannerComponentsNavigationDto` you use separate lines  annotation and value

Reply:
or vice versa